programStructure/Looping a triangle.js

```diff
@@ -0,0 +1,18 @@
+/*
+Write a loop that makes seven calls to console.log to output the following triangle:
+
+#
+##
+###
+####
+#####
+######
+#######
+It may be useful to know that you can find the length of a string by writing .length after it.
+let stringForTriangle = "";
+*/
+
+for(let i = 0; i < 7; i++)
+{
+  console.log(stringForTriangle += "#");
+}
```

Comment:
Please, run your code in order to test if it works before commit any changes (because it doesn't )

You have to use **'while and do loops'**  to finish this task instead of 'for' loop